apps/dashboard/src/@/analytics/report.ts

```diff
@@ -435,11 +435,8 @@ export function reportPaymentLinkCreated(properties: {
  * ### Who is responsible for this event?
  * @greg
  */
-export function reportPaymentLinkBuySuccessful(properties: {
-  linkId: string;
-  clientId: string;
-}) {
-  posthog.capture("payment link buy successful", properties);
+export function reportPaymentLinkBuySuccessful() {
+  posthog.capture("payment link buy successful");
```

Comment:
Don't we want the payment details in here?

Reply:
No - its not important